### PR TITLE
Cuts sci budget income to a tenth of node price and removes roundstart free money

### LINKED ISSUE #41801: rd mad dosh isnt even intended
http://take.ms/peQyMP this happens roundstart by triggering every node and giving you like 30K for free
i tested more on local but i didnt get mad dosh like on the live server, can we just adminlog every adjust_money() on live server for a while to find where the  other economy  bugs are?

### PR DESCRIPTION
**Pull request by xenon-difluoride**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: optional name here
balance: the RD is no longer the Monopoly Man
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
image
Currently the RD can spam guns very easily, this is a quick patch for it until someone manages to make money = points
closes #41801

### CODE REVIEW

**Comment by evsey9** [comment]
NO
MY MONEY

File: code/modules/research/techweb/_techweb.dm [hunk 1]
```diff
@@ -217,7 +217,7 @@
 	if(!istype(src, /datum/techweb/admin))
```
Comment (XDTM) [inline code/modules/research/techweb/_techweb.dm]:
```suggestion
	if(!istype(src, /datum/techweb/admin) && !node.starting_node)
```

Or just do this and exclude starting nodes

**Comment by Tlaltecuhtli** [comment]
this doesnt close the issue as it only fixes 1 known bug and also the having /10 of income from techwebs doesnt fix shit

File: code/modules/research/techweb/_techweb.dm (hunk 1 above)
Comment (AlexTheSysop42) [reply code/modules/research/techweb/_techweb.dm]: It wouldn't fix the problem of future nodes giving mad dosh but it might be a good idea to have as well

**Comment by Barhandar** [comment]
The point of sci budget is that research points == cash (because economy is garbage), making income different than 100% of the node price is therefore against the point.

**Comment by AlexTheSysop42** [comment]
currently if you take out money, add money, or spend points it won't change the other, if someone manages to make them convert then 1:1 would be fine
the issue now is that RD gets mad dosh for free and can spam guns 30 seconds into the round

also @Tlaltecuhtli the RD budget is not super high with this change applied, it was probably a result of mad dosh per node + free roundstart nodes

**Comment by improvedname** [comment]
the joke is that it DOESNT take techweb points.

**Comment by Tlaltecuhtli** [comment]
@AlexTheSysop42 the whole issue is that it generates more than intended trough bugs n glitches, if it worked like it should 250$/ node means 35000$ after 1h round which is fine and balanced. Turning it to 3500$ in 1h is dumb remove the /10 change

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm leaving it open if someone wants to testmerge as a bandaid but i'm doing a read this evening and will prob PR the actual fix (I have a hunch already)

**Comment by AlexTheSysop42** [comment]
@Tlaltecuhtli this MAKES it 35k after one round rather than 350k
have you even tried it with this

@ExcessiveUseOfCobblestone I put the math here because I thought the bounty var was based on node price or something

**Comment by evsey9** [comment]
Easier fix would be to just remove economy

**Comment by Tlaltecuhtli** [comment]
the current bounty is 250$ not 2500$ @AlexTheSysop42

**Comment by AlexTheSysop42** [comment]
The current amount you get is 2500$

**Comment by ExcessiveUseOfCobblestone** [comment]
Strange, it should be 250 since well that's what the var is lol. Are you sure you're not also calculating the science paycheck (aka xenobio bonuses)? They're all multiples of 10 so it's quite possible it totaled 2500 with the 250 included.

**Pull request closed without merging**